c_binding/include/libstoragemgmt/libstoragemgmt_capabilities.h

```diff
@@ -115,6 +115,32 @@ typedef enum {
     LSM_CAP_VOLUME_THIN = 55,
     /**^ Thin provisioned volumes are supported */
 
+    /** Query battery is supported */
+    LSM_CAP_BATTERIES = 56,
+
+    /** Query volume ram cache info is supported */
+    LSM_CAP_VOLUME_CACHE_INFO = 57,
+
+    /** Set volume physical disk cache */
+    LSM_CAP_VOLUME_PHYSICAL_DISK_CACHE_SET = 58,
```

Comment:
Change these capability names to _UPDATE instead of _SET also, keeping in line with the naming change for the operations that can change controller system settings. There are several that would need to change to _UPDATE here.